.github/workflows/spec-parser.yml

```diff
@@ -17,11 +20,11 @@ jobs:
       contents: write
 
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
       - name: Set up Python
-        uses: actions/setup-python@v4
+        uses: actions/setup-python@0a5c61591373683505ea898e09a3ea4f39ef2b9c # v5.0.0
         with:
-          python-version: "3.11"
+          python-version: "3.12"
       - name: get spec-parser
```

Comment:
This section where we run the spec-parser should be deleted - the current output formats don't currently work with the latest version.

@licquia will add this back to the spec-parser repo